src/Ninja/G1CP/Content/Fixes/Gamesave/fix0057_CastleWallTexture.d

```diff
@@ -1,38 +1,39 @@
 /*
  * #57 Misplaced texture near Gravo
  */
-func int G1CP_0057_CastleWallTexture() {
-    var int vobPtr; vobPtr = G1CP_FindVobByPosF(-5893.91455, -664.270386, 78.9339142, G1CP_0057_CastleWallTextureCheck);
-    if (vobPtr) {
-        G1CP_MoveVobToPosF(vobPtr, -5902.01807, -664.270386, 82.8507462);
-        return TRUE;
-    };
-    return FALSE;
-};
+
+/*
+ * Make the positions available to the functions below
+ */
+const float G1CP_0057_CastleWallTexture_OriginalPos[3] = {-5893.91455, -664.270386, 78.9339142};
+const float G1CP_0057_CastleWallTexture_CorrectedPos[3] = {-5902.01807, -664.270386, 82.8507462};
 
 /*
  * Callback function to check the found VOBs: Check if the VOB's visual is a decal
  */
 func int G1CP_0057_CastleWallTextureCheck(var int vobPtr) {
     var zCVob vob; vob = _^(vobPtr);
-    const int zCDecal_classDef = 8861528; //0x873758
+    const int zCDecal_classDef = 8861528; // 0x873758
+
     return (MEM_GetClassDef(vob.visual) == zCDecal_classDef);
 };
 
+/*
+ * Apply the fix
+ */
+func int G1CP_0057_CastleWallTexture() {
+    return G1CP_ChangeVobLocation(_@(G1CP_0057_CastleWallTexture_OriginalPos),
+        _@(G1CP_0057_CastleWallTexture_CorrectedPos), G1CP_0057_CastleWallTextureCheck);
```

Comment:
I still have to get used to this way of breaking lines. I agree that it looked also odd before. What do you think about breaking after and before the "(" and ")" as well in such a case? That would resemble the OTB-like bracket style used in the project.

```suggestion
    return G1CP_ChangeVobLocation(
        _@(G1CP_0057_CastleWallTexture_PosOld), _@(G1CP_0057_CastleWallTexture_PosNew),
        G1CP_0057_CastleWallTextureCheck
    );
```
I hope line wrapping doesn't make this example look too weird.

Reply:
Great compromise! I think that's how styles of other languages do it, too, right? It looks familiar from black/ruff in Python, maybe also in Prettier for js/ts?